react/features/security/components/security-dialog/SecurityDialogButton.js

```diff
@@ -64,9 +70,13 @@ class SecurityDialogButton extends AbstractButton<Props, *> {
 function mapStateToProps(state: Object) {
     const { locked } = state['features/base/conference'];
     const { lobbyEnabled } = state['features/lobby'];
+    const enabledFlag = getFeatureFlag(state, SECURITY_OPTIONS_ENABLED, true);
+    const enabledLobbyModeFlag = getFeatureFlag(state, LOBBY_MODE_ENABLED, true);
+    const enabledMeetingPassFlag = getFeatureFlag(state, MEETING_PASSWORD_ENABLED, true);
 
     return {
-        _locked: locked || lobbyEnabled
+        _locked: locked || lobbyEnabled,
+        visible: enabledFlag || (enabledLobbyModeFlag && enabledMeetingPassFlag)
```

Comment:
You probably intended to use the logic operators in revers: `visible: enabledFlag && (enabledLobbyModeFlag || enabledMeetingPassFlag)`

Note also that lobby might be hidden from base/config or not available (check mapStateToProps in SecurityDialog). We should check for that too here.

Reply:
Updated condition.